requirements.txt

```diff
@@ -28,6 +28,7 @@ pyink
 pre-commit
 pytype
 pillow>=11.1.0
+qwix@git+https://github.com/google/qwix.git
```

Comment:
This dependency should also be in `constraints_gpu` and `requirements_with_jax_stable_stack_0_6_1_pipreqs.txt`